Address review comments for InnerSource Hackathon pattern (#721)
gruetter/innerSourceTest: patterns/1-initial/innersource-hackathon.md

> The senior leadership believes in InnerSource and wants to drive it throughout the company. The engineers who are familiar with open source principles and/or understand the benefits of InnerSource are the early adopters. There is success with these initial pilot project and teams. Now the next step is to drive it across the company. There might be reluctance from engineering teams due to various factors like:

Reviewer: ```suggestion
The senior leadership believes in InnerSource and wants to drive it throughout the company. The engineers who are familiar with open source principles and/or understand the benefits of InnerSource are the early adopters. There is success with these initial pilot project and teams.

Now the next step is to drive it across the company. There might be reluctance from engineering teams due to various factors like:
```

Splitting this into 2 paragraphs to make it easier to read.

> * not familiar with InnerSource and not aware about it

Reviewer: "not familiar" and "not aware" sounds like saying the same thing twice now. I am not sure though.

I have also asked in [the thread in slack](https://innersourcecommons.slack.com/archives/C2EFRTS6A/p1728330592897019?thread_ts=1727901171.632369&cid=C2EFRTS6A) if anybody has a better proposal.

Author: Sorry for the delay in addressing the comments. I have updated this point to "not familiar with InnerSource or open source practices", as suggested by Sally Deering in the slack thread.

* not enough time to prioritize InnerSource, given the regular work deliverables

> * relunctance to changing ways of working when everything works well already

Reviewer: ```suggestion
* reluctance to changing ways of working when everything works well already
```

Fixing typo

> * unclear ROI for the upfront setup innersourcing takes

Reviewer: ```suggestion
* unclear return on investment for the upfront setup costs that an InnerSource project takes
```

We try to stick to the spelling InnerSource, wherever possible. See [explanation](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/meta/innersource-spelling.md).

Also I removed an acronym and elaborated on the sentence a bit
